src/buildForms.js

```diff
@@ -0,0 +1,68 @@
+const { Buffer } = require("buffer");
+const { v4: uuidv4 } = require("uuid");
+const { hideBin } = require("yargs/helpers");
+const yargs = require("yargs/yargs");
+
+const {
+  clearDistFolder,
+  createFormTtlFile,
+  createResourceMigrationFile,
+  bundleForms,
+  generateTimestamp,
+} = require("./utils.js");
+
+const argv = yargs(hideBin(process.argv)).option("timestamp", {
+  alias: 't',
+  default: null,
+  defaultDescription:
+    "It will generate a new timestamp based on the current time",
+  description:
+    "The timestamp that should be used for the file and folder names. Format: `YYYYMMDDHHmmss`",
+  string: true,
+}).version(false).argv;
+
+const defaultFileGraph = "http://mu.semte.ch/graphs/public";
+const projects = [
+  {
+    name: "app-digitaal-loket",
+  },
+  {
+    name: "app-toezicht-abb",
+    fileGraph:
+      "http://mu.semte.ch/graphs/organizations/141d9d6b-54af-4d17-b313-8d1c30bc3f5b",
+  },
+  {
+    name: "app-meldingsplichtige-api",
+  },
+  {
+    name: "app-public-decisions-database",
+    fileGraph:
+      "http://mu.semte.ch/graphs/access-for-role/PubliekeBesluitendatabank-BesluitendatabankLezer",
+  },
+];
+
+clearDistFolder();
+
+let bundledFormOutput = bundleForms();
+let fileSize = Buffer.byteLength(bundledFormOutput);
+let now = new Date();
+let timestamp = argv.timestamp || generateTimestamp(now);
+let fileName = `${timestamp}-forms.ttl`;
+let uuid = uuidv4();
+
+projects.forEach((project) => {
+  createFormTtlFile({ project, fileName, bundledFormOutput });
+
+  let targetGraph = project.fileGraph ?? defaultFileGraph;
+
+  createResourceMigrationFile({
+    project,
+    timestamp,
+    targetGraph,
+    fileName,
+    fileSize,
+    dateCreated: now,
```

Comment:
Yea, I was thinking about that indeed. I'm not sure to ask for date information through terminal args though. Any suggestions?

I figured it wasn't worth the work since this was only estimated as 5 SP :smile: so I left it for now.

Reply:
I can replace the `--timestamp` option with a `--date` instead and enter an iso date which we parse with https://date-fns.org/v2.28.0/docs/parseISO?